config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
-    get "/help",    to: "static_pages#help"
-    get "/about",   to: "static_pages#about"
-    get "/contact", to: "static_pages#contact"
-    resources :users, only: %i(show new create)
+    get "/help",      to: "static_pages#help"
```

Comment:
Sau dấu phẩy chỉ 1 space -> self check

```suggestion
    get "/help", to: "static_pages#help"
```